Simulations/README.md

```diff
@@ -54,90 +54,87 @@ To run the default set of FITS files, as described in [Data Product Summary](#da
 
 
 ```
-> ./run_recipes.py --doCalib=1 --sequence=1 --doStatic
-> md5sum -c checksums.dat | grep -v OK
+> /runESO.sh
```

Comment:
```suggestion
> ./runESO.sh
```

Reply:
This error is still there; each of these suggestions have a button "Commit suggestion" next to it, so if you agree with the suggested change, you can just press the button and it is done. (Before merging.)